### PR TITLE
Noetic version

### PR DESCRIPTION
**Pull request by artivis**
Port snap to ROS Noetic.

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -33,203 +33,266 @@ apps:
...
-      - ros-jazzy-nao-lola-sensor-msgs
-      - ros-jazzy-ouster-sensor-msgs
-      - ros-jazzy-ros-babel-fish-test-msgs
-      - ros-jazzy-ros2-socketcan-msgs
-      - ros-jazzy-rosbag2-performance-benchmarking-msgs
-      - ros-jazzy-rosbridge-test-msgs
-      - ros-jazzy-service-msgs
-      - ros-jazzy-soccer-geometry-msgs
-      - ros-jazzy-soccer-model-msgs
+      - ros-noetic-abb-egm-msgs
```
Comment (Guillaumebeuzeboc) [inline snap/snapcraft.yaml:55]: Isn't it missing the services definition packages?

Comment (artivis) [reply snap/snapcraft.yaml:55]: Most services aren't in separate packages as msgs can be. I've added the std srvs tho. The snap could be converted to using ROS content-sharing in futur iteration in order to have access to more srvs without pulling half of the ROS stack.

**Review by Guillaumebeuzeboc — APPROVED** [review]

**Comment by Guillaumebeuzeboc** [comment]
I think a change in `local/launch` is missing

**Comment by artivis** [comment]
> I think a change in `local/launch` is missing

Good catch, pushed it.

**Review by Guillaumebeuzeboc — APPROVED** [review]

**Review by Guillaumebeuzeboc — APPROVED** [review]

**Pull request merged**